clients/double_click_bid_manager/lib/google_api/double_click_bid_manager/v1/connection.ex

```diff
@@ -26,6 +25,6 @@ defmodule GoogleApi.DoubleClickBidManager.V1.Connection do
       # View and manage your reports in DoubleClick Bid Manager
       "https://www.googleapis.com/auth/doubleclickbidmanager"
     ],
-    otp_app: :google_api_double_click_bid_manager,
-    base_url: "https://www.googleapis.com/doubleclickbidmanager/v1"
+    otp_app: :google_api_doubleclickbidmanager,
```

Comment:
This seems incorrect

Reply:
I've noticed this in a few places also. Does it actually matter?